qiskit_metal/tests/test_toolbox_metal.py

```diff
@@ -324,10 +348,532 @@ def test_toolbox_metal_cross(self):
         self.assertEqual(math_and_overrides.cross(my_array_1, my_array_2), -6)
 
     def test_toolbox_metal_aligned_pts(self):
-        #   """Test functionality of aligned_pts in toolbox_metal.py."""
+        """Test functionality of aligned_pts in toolbox_metal.py."""
         self.assertTrue(math_and_overrides.aligned_pts([0, 1, 2]))
         self.assertFalse(math_and_overrides.aligned_pts([7, 1, 2]))
 
+    def test_toolbox_metal_determine_larger_box(self):
+        """Test functionality of determine_larger_box in toolbox_metal.py."""
+        self.assertEqual(
+            bounds_for_path_and_poly_tables.determine_larger_box(
+                0.0, 0.0, 11.0, 11.0, (0.0, 0.0, 10.0, 10.0)),
+            (0.0, 0.0, 10.0, 10.0))
+
+    def test_toolbox_metal_get_bounds_of_path_and_poly_tables(self):
+        """Test functionality of get_bounds_of_path_and_poly_tables in toolbox_metal.py"""
+        ls_file_path = ("./qiskit_metal/tests/test_data/planar_chip.txt")
+        multiplanar_design = MultiPlanar(metadata={},
+                                         overwrite_enabled=True,
+                                         layer_stack_filename=ls_file_path)
+        conn_pads = dict(connection_pads=dict(coup1=dict(loc_W=-1, loc_H=1),
+                                              coup2=dict(loc_W=1, loc_H=1)))
+        q1 = TransmonPocket6(multiplanar_design,
+                             "Q1",
+                             options=dict(**conn_pads, chip="q_chip", layer=2))
+        qc1 = QuadCoupler(multiplanar_design,
+                          "qc1",
+                          options=dict(pos_x="0mm",
+                                       pos_y="-0.08mm",
+                                       pad_width="120um",
+                                       pad_height="30um",
+                                       cpw_stub_height="250um",
+                                       chip="c_chip",
+                                       layer=1))
+
+        self.assertIsInstance(
+            BoundsForPathAndPolyTables(
+                multiplanar_design).get_bounds_of_path_and_poly_tables(
+                    False, [], 1, 0, 0), tuple)
```

Comment:
There are many things returned in the tuple.  
Could you check the tuple that the total number of items has not changed, and possibly check the type?

 Returns:
            tuple[tuple[float, float, float, float], pd.DataFrame, pd.DataFrame, bool, Union[None,set]]:
                                    tuple: minx, miny, maxx, maxy values based
                                                    on either total chip size or box_plus_buffer.
                                                    In xy plane, the box_for_xy_bounds will be limited
                                                    by the chip_bounds_xy if box_for_xy_bounds is larger
                                                    than chip_bounds_xy.
                                    pd.DataFrame: The path and poly dataframes concatenated for qcomp_ids
                                    pd.DataFrame: The path, poly, junction dataframes concatenated for qcomp_ids
                                    bool: If there is a key in design with same chip_name as in layer_stack
                                    Union[None,set]: Either None if the names don't match,
                                                    or the set of chip_names that can be used.